packages/core/src/components/Form/FormComponents.css

```diff
@@ -39,14 +31,14 @@
   display: block;
   composes: fontSmallIi from '../../globals/typography.css';
   color: var(--color-greyDark);
-  font-weight: var(--fontweight-demi);
+  composes: demi from '../../globals/modifiers.css';
```

Comment:
We've got a lot of overlap between using `var`s, importing from `globals/modifiers.css` and importing from `globals/typography.css`

Reply:
it boils down to how css modules work...We'll probably find something better in the long run